autopush_rs/src/client.rs

```diff
@@ -730,6 +730,19 @@ where
             .send()
             .ok();
 
+        // If there's any notifications in the queue, move them to our unacked direct notifs
+        webpush.rx.close();
+        let rx_iter = webpush.rx.wait();
```

Comment:
Initially this threw me for a bit of a spin triggering the "oh no the `wait` method shouldn't be used on the event loop!" but after thinking about it I think it's fine due to the `close` above. Mind throwing a comment here saying "no worries, this won't block due to the `close` above"?

Reply:
Ah, yes, added a callout.